app/controllers/placements_controller.rb

```diff
@@ -1,6 +1,59 @@
 # frozen_string_literal: true
 
 class PlacementsController < ApplicationController
+
+    before_action :authenticate_user!
+    require 'roo'
+    
+    def new
+        @placement = Placement.new
+        @licensees = Licensee.all
+        @services = Service.all
+    end
+    
+    def create
+        Placement.create(
+            name: params[:placement][:name],
+            licensee_id: params[:placement][:licensee_id],
+            address: params[:placement][:address],
+            city: params[:placement][:city],
+            state: params[:placement][:state],
+            zip: params[:placement][:zip],
+            county: params[:placement][:county],
+            phone: params[:placement][:phone],
+            gender: params[:placement][:gender],
+            beds: params[:placement][:beds],
+        )
+        redirect_to placements_path
+    end
+    
+    # licensee_id: params[::placement][:licensee_id]
+    # service_id: params[:placement][:service_id],    
+    
+    def index
+        # search = "%#{params[:search]}%"
+        # @placements = Placement.where("name LIKE ? OR county LIKE ?" , search, search)
+
+        @q = Placement.ransack(params[:q])
+        @placements = @q.result(distinct: true)
+
+        respond_to do |format|
+            format.xlsx {
+                response.headers[
+                'Content-Disposition'
+                ] = "attachment; filename=placements.xlsx"
+            }
+            format.html { render :index }
+        end
+    end
+    
+    def show
+        @placement = Placement.find(params[:id])
+        @licensee = Licensee.find(@placement.licensee_id)
+        @comment = Comment.new(placement_id:@placement.id)
+        @comments = @placement.comments.collect
+    end
+
   before_action :authenticate_user!
   before_action :authorize_admin, only: %i[new edit]
```

Comment:
This controller also seems to contain a full duplicate of all of the methods. Please correct this file so that it contains only one version.